testing/travis-fork-check.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import os
+import subprocess
+import sys
+
+def check_travis_repo_slug():
+    if 'encrypted_6a9d32f3e0bd_key' not in os.environ:
+        sys.stderr.write("Decryption disabled. Setting MANTL_CI_FORK_CHECK to 1")
+        print("1")
+    else:
+        sys.stderr.write("Decrypting OS ssh key")
+        cmd = ["openssl",
```

Comment:
If the approach in the travis file is good, this whole file can be deleted.
> On May 12, 2016, at 14:33, Brian Hicks <notifications@github.com> wrote:
> 
> In testing/travis-fork-check.py <https://github.com/CiscoCloud/mantl/pull/1415#discussion_r63082662>:
> 
> > @@ -0,0 +1,27 @@
> > +#!/usr/bin/env python
> > +from __future__ import print_function
> > +import os
> > +import subprocess
> > +import sys
> > +
> > +def check_travis_repo_slug():
> > +    if 'encrypted_6a9d32f3e0bd_key' not in os.environ:
> > +        sys.stderr.write("Decryption disabled. Setting MANTL_CI_FORK_CHECK to 1")
> > +        print("1")
> > +    else:
> > +        sys.stderr.write("Decrypting OS ssh key")
> > +        cmd = ["openssl",
> why is this script still decrypting the key? Isn't that happening in line 38 of travis.yml now?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/CiscoCloud/mantl/pull/1415/files/89c1b3d3c65fa0ac9d1a205342ee47aa9e629995#r63082662>